Enable Email Confirmation

holenet opened the pull request:
If a student has signed up, the student cannot sign in immediately.
When a student signed up, the backend will send a confirmation mail to the student's SNU account of  automatically.
Once the student click the link in the confirmation mail, since then, the student can sign in.

I made a google account for sender of the email confirmation. Please see `settings.py`.

**BUT**, I wonder if we should disable this feature because it is expected to be troublesome when development and test.
I can make a student able to sign in even if the student has not confirmed email account, by changing only one line of the code.
So, please give me your opinions.
I'll put them together and modify the code.

holenet committed:
enable email confirmation

Gamjae commented:
I agree. If e-mail confirmation makes our development harder, we'd better not to send confirm mail.
Also, it is not one of main functions we have to implement.

lameinthebox reviewed (approved):
Well done!
I think we can leave email configuration active since we've finished implementing signup in frontend. (We can signup to make test ids once and use it for a further implement, so not that tiresome, I suppose?) But either way is fine for me.

holenet committed:
Disable email confirmation for development convenience

A student is able to sign in without email confirmation.
Still, the confirmation email will be send to the email account of the student.
So, it would be better to input your actual SNU account when you test sign up feature.

Tommybro commented:
When we develop frontend, we need to make new user frequently. So I think email validation should be used after implementing all other functions.

holenet commented:
I omitted inactivation of student when the student just has created.
So, we can use that student without any email confirmation.
But, the confirmation mail will be send to the account typed when sign up.
For the specification, please refer this [commit](https://github.com/SWPP/ttrs-backend/pull/62/commits/b6987ec4b7e6bb049ca61520cdc13fb5c3018471)

Tommybro commented:
ok
